starter/script.js

```diff
@@ -1 +1,73 @@
-// put code here!
+function startGame() {
+var name = prompt('What is your NAME?');
+'hello '+ name + '!';
+var Class = prompt('choose WARRIOR or ROGUE');
+  'you have chosen' + Class + '!';
```

Comment:
could `console.log` or `alert` this string so the user has a visual interface with your program